docs/admin/logging/journald.md

```diff
@@ -42,6 +42,9 @@ You can set the logging driver for a specific container by using the
 Users can use the `--log-opt NAME=VALUE` flag to specify additional
 journald logging driver options.
 
+Users can use the `--log-opt journal-gateway-address=host:port` option to
+specify the `systemd-journal-gatewayd` address where the driver connects to.
+
 ### tag
```

Comment:
This reads to me as indicating that the gateway is also used as the destination when writing the logs, but that's a function of _systemd-journal-remote_.  Any plans to augment the log driver to log directly to a remote server over HTTP or HTTPS?